rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -719,43 +719,24 @@ def _wrap_paginated_response(
         link = cls._build_url(request, head=head)
 
         paging_response = response['paging']
```

Comment:
This behavior is missing from `master` (I think because it would've been a pain to implement), but the response `"link"` should include explicit `start` and `limit` queries that reflect any validator imposed defaults or maximums. If `start` and `limit` are added to the `ClientPagingResponse` this becomes trivial.

Simply move line 721 before line 719, then change line 719 to:
```
link = cls._build_url(request, head=head, start=paging_response['start'], limit=paging_response['limit'])
```